ember/ember-cli-build.js

```diff
@@ -10,6 +10,10 @@ module.exports = function(defaults) {
       extensions: ['css', 'js'],
       exclude: ['cesium'],
     },
+    minifyJS: {
+      enabled: EmberApp.env() !== 'production',
```

Comment:
Interesting. I just ran a test locally by deleting the `enabled` flag completely.

The result was that the `exclude` setting was completely ignored! Only when the `enabled` key is present - it seems - is the `exclude` setting honored.

This is perhaps a bug with ember-cli or at the very least a documentation issue.

I would actually like for someone else to confirm this for me as this is kinda unbelievable. :)

# Summary -> You need the `enabled` key for any settings to take effect, I think.